api/docs/release.dox

```diff
@@ -190,6 +190,8 @@ Further non-compatibility-affecting changes include:
  - Added instr_is_exclusive_load().
  - Added XINST_CREATE_store_pair() and XINST_CREATE_load_pair().
  - Added instr_replace_reg_resize() and opnd_replace_reg_resize().
+ - Added DRX_COUNTER_REL_ACQ flag to optionally enable release-acquire semantics for
```

Comment:
Use a `#` prefix (so `#DRX_COUNTER_REL_ACQ`) to have Doxygen generate an html link (`()` does it for functions but need `#` for enums) as well as serving as a sanity check that there's no typo (Doxygen will generate an error if there is no such symbol with that name).

Reply:
I see. Done.